Documentation/cassandra-cluster-crd.md

```diff
@@ -79,7 +81,6 @@ Rack Settings:
 * `resources`: Defines the CPU and RAM resources for the Cassandra Pods.
 * `placement`: Defines the placement of Cassandra Pods. Has the following subfields:
     * [`nodeAffinity`](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
-    * [`nodeAntiAffinity`](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
```

Comment:
I think it would be better to add `nodeAntiAffinity` to the placement type and document it per storage backend if it is available (e.g., nodeAffinity and/or nodeAntiAffinity is not available for Ceph mons). Though this could be okay to add in another PR.

Reply:
The thing is, nodeAntiAffinity doesn't exist as a field in the Kubernetes API. 
It was a mistake on my part, I thought it existed like podAntiAffinity, but it doesn't.